keyboards/adalyn/keymaps/vial/keymap.c

```diff
@@ -0,0 +1,49 @@
+ /* Copyright TJ Campie
```

Comment:
VIAL is not supported in the main repo, and the vial config and keymap should be removed.